docs/orgs.md

```diff
@@ -70,6 +70,29 @@ This will create the organization data in a file `group-<snyk_group_id>-bitbucke
 This will create the organization data in a file `group-<snyk_group_id>-bitbucket-cloud-orgs.json`
 
 
+## Azure
```

Comment:
Please can you also link to this in the table of contents above? And while you are there fix the spelling of Bitbucket items 🙏 No need to the `-`
<img width="725" alt="CleanShot 2022-06-20 at 13 54 58@2x" src="https://user-images.githubusercontent.com/2911613/174606351-60600baf-3484-488c-94ed-5ac7ed072fc8.png">